source/wp-content/themes/wporg-developer-2023/inc/block-hooks.php

```diff
@@ -52,26 +52,20 @@ function get_block_content_by_home_url( $block_content, $replacement_home_url =
 }
 
 /**
- * Filters the article meta block and conditionally removes it.
+ * Filters an article meta block and conditionally removes it.
  *
  * @param string $block_content
  * @param array  $block
  * @return string
  */
-function filter_article_meta_block( $block_content, $block ) {
-	if ( 'wporg/article-meta-github' === $block['blockName'] ) {
-		$github_handbooks = array(
-			'wpcs-handbook',
-			'blocks-handbook',
-			'rest-api-handbook',
-			'cli-handbook',
-			'adv-admin-handbook',
-		);
+function filter_article_meta_block( $block_content, $block ) {	
+	// Not all handbooks come from GitHub.
+	$local_handbooks = array( 'plugin-handbook', 'theme-handbook' );
```

Comment:
Reverse this logic based on the assumption that new handbooks added would be GitHub based, and this logic would not have to be changed. GitHub handbooks allow community updates.